app/code/Magento/Sales/Model/Order/Email/SenderBuilder.php

```diff
@@ -86,6 +103,8 @@ public function sendCopyTo()
 
         if (!empty($copyTo) && $this->identityContainer->getCopyMethod() == 'copy') {
             foreach ($copyTo as $email) {
+                $this->resetTransportBuilder();
```

Comment:
For both occurrences: we must reset only after `TransportBuilder` is used for a first time.

Reply:
I've requested guidance on this